landscape/client/watchdog.py

```diff
@@ -535,6 +537,22 @@ def get_enabled_daemons(self):
             daemons.append(Manager)
         return daemons
 
+    def convert_arg_to_bool(self, value: str) -> bool:
```

Comment:
Can we move this into `landscape/client/deployment.py` and make it a function? I don't think it really needs to be a method on a particular configuration type. It's more generic than that.

We should also add unit tests specifically for this function.

If you'd like to mimic Python's configparser here, you can use these values:

```python
# Case-insensitive values to use when parsing booleans.
TRUTHY_VALUES = {"true", "yes", "y", "1", "on"}
FALSY_VALUES = {"false", "no", "n", "0", "off"}
```

We had to implement something similar for server in `canonical/lib/config.py`. You can use the unit tests there for reference on these unit tests, too.